### PR TITLE
Ref: cleaning CI and removing travis

### PR DESCRIPTION
**Pull request by marandaneto**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [X] Refactoring

## :scroll: Description
Ref: cleaning CI and removing travis

## :bulb: Motivation and Context
We don't need Travis if using GH actions.
Moving Codecov to GH actions
Separating check formatting to its own workflow

## :pencil: Checklist

- [X] I reviewed submitted code
- [ ] I added tests to verify changes
- [X] All tests passing
- [X] No breaking changes

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -77,6 +67,13 @@ jobs:
           name: packages
           path: ./*/build/distributions/*.zip
 
+      - name: Upload coverage to Codecov
+        # We need coverage data from only one the builds
+        if: ${{ matrix.os }} == ubuntu-latest
```
Comment (marandaneto) [inline .github/workflows/build.yml:72]: @maciejwalkowiak I just copied from `Archive packages` but this seems to not work, its executed on mac-os too

Comment (maciejwalkowiak) [reply .github/workflows/build.yml:72]: @bruno-garcia set this up for Archive Packages. Perhaps take a look here: https://github.community/t/what-is-the-correct-if-condition-syntax-for-checking-matrix-os-version/16221/3

Comment (marandaneto) [inline .github/workflows/build.yml:72]:
```suggestion
        if: runner.os == 'Linux'
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -52,23 +50,15 @@ jobs:
...
-      # - name: Stop Gradle
-      #   uses: eskatos/gradle-command-action@v1
-      #  with:
-      #    arguments: --stop
       - name: Make stop
         run: make stop
+
       - name: Archive packages
         # We need artifacts from only one the builds
         if: ${{ matrix.os }} == ubuntu-latest
```
Comment (marandaneto) [inline .github/workflows/build.yml:64]:
```suggestion
        if: runner.os == 'Linux'
```

File: .github/workflows/build.yml (hunk 1 above)
Comment (marandaneto) [reply .github/workflows/build.yml]: `runner.os == 'Linux'` instead works.

**Comment by marandaneto** [comment]
Screenshot 2020-11-14 at 00 04 03

**Pull request merged**